docs/api/apiv3/components/schemas/user_preferences_model.yml

```diff
@@ -17,30 +17,11 @@ example:
   timeZone: "Europe/Berlin"
   warnOnLeavingUnsaved: true
   notifications:
-    - channel: "mail"
-      watched: false
-      involved: true
-      mentioned: false
-      all: false
-      workPackageCommented: false
-      _links:
-        project:
-          href: null
-    - channel: "in_app"
-      watched: true
-      involved: true
-      mentioned: true
-      all: false
-      workPackageCommented: false
-      _links:
-        project:
-          href: null
-    - channel: "mail_digest"
-      watched: true
-      involved: true
-      mentioned: true
-      all: true,
-      workPackageCommented: true
-      _links:
-        project:
-          href: null
+    watched: false
+    involved: true
+    mentioned: false
+    all: false
```

Comment:
The `all` is to be removed and, from the previous PR,

```
  newsAdded: false,
  newsCommented: false,
  documentAdded: false,
  forumMessages: false,
  wikiPageAdded: false,
  wikiPageUpdated: false,
  membershipAdded: false,
  membershipUpdated: false
```

is to be added

And since I forgot to do so, it would be very kind of you if you were to add:

```
  workPackageCommented: false,
  workPackageProcessed: false,
  workPackagePrioritized: false,
  workPackageScheduled: false
```